packages/sdks/src/blocks/tabs/tabs.lite.tsx

```diff
@@ -0,0 +1,77 @@
+import { For, Show, useStore, useTarget } from '@builder.io/mitosis';
+import Blocks from '../../components/blocks/blocks.lite.jsx';
+import type { BuilderBlock } from '../../types/builder-block.js';
+import type { TabsProps } from './tabs.types.js';
+
+export default function Tabs(props: TabsProps) {
+  const state = useStore({
+    activeTab: props.defaultActiveTab ? props.defaultActiveTab - 1 : 0,
```

Comment:
this should be a `getter` so that it reacts to props changes (like changes in the visual editor)

Reply:
@samijaber sure! if we turn it to a getter should we have another state for whats active, because we are updating it when we click on different tabs and a getter is a read-only property

Reply:
oh right. nevermind then, it's good as-is